numpy diaganol

```diff
@@ -0,0 +1 @@
+
```

Comment:
Also, you still haven't reverted the changes on unrelated files. Your PR should only affect `indexing_like_operations.py` and `test_indexing_like_operations.py`.

Reply:
Maybe I missed it. I tried to revert all the changes. Please tell me which one.

Reply:
All changes in files other than `indexing_like_operations.py` and `test_indexing_like_operations.py`.
You can see [here ](https://github.com/unifyai/ivy/pull/6652/files)that there are 20 modified files, while it should be 2 files.

Reply:
> All changes in files other than `indexing_like_operations.py` and `test_indexing_like_operations.py`. You can see [here ](https://github.com/unifyai/ivy/pull/6652/files)that there are 20 modified files, while there should only be 2 files.

How I revert those changes? I made changes in array_module.py regarding backend, which I removed already. But after updating (merging), the commit show all the files including the two relevant ones.  So reset maybe a option or not. In other case as you said creating a new PR. Then should I delete this PR first and then create a new one.

Reply:
I would recommend you make a new PR where you only include changes to `indexing_like_operations.py` and `test_indexing_like_operations.py` (be careful what files you commit when making it). 
Comment the link of the new PR here and I will take care of it. You don't need to delete this PR, I will just close it.

Reply:
Most of the unwanted files have been removed from the PR. Now you have 6 changed files instead of the 2 we want. This is the only important thing remaining to finish and merge your PR.